components/puppeteer/package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "@pipedream/puppeteer",
-  "version": "0.0.2",
+  "version": "0.1.0",
   "description": "Pipedream Puppeteer Components",
-  "main": "dist/app/puppeteer.app.mjs",
+  "main": "puppeteer.app.mjs",
   "keywords": [
     "pipedream",
     "puppeteer"
   ],
-  "files": [
-    "dist"
-  ],
   "homepage": "https://pipedream.com/apps/puppeteer",
   "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
```

Comment:
`@pipedream/platform` is missing here (since we're using its `ConfigurationError`)